dev-support/create-release/release-util.sh

```diff
@@ -20,6 +20,9 @@ DRY_RUN=${DRY_RUN:-1} #default to dry run
 DEBUG=${DEBUG:-0}
 GPG=${GPG:-gpg}
 GPG_ARGS=(--no-autostart --batch)
+if [ -n "${GPG_KEY}" ]; then
+  GPG_ARGS=("${GPG_ARGS[@]}" --local-user "${GPG_KEY}")
```

Comment:
Again, to my previous comments, I'm surprised this script identifies GPG identity via an email address/`--local-user`. I expected that we'd manage everything via a key fingerprint/`--default-key`.

Reply:
it doesn't identify the gpg identity necessarily via email address. `--local-user` takes the name of a key, the same as `--default-key` does. that name can be a keyid, email address, or fingerprint.  the `--local-user` option is defined as "I specifically want you to use this identity to sign" and overrides any given default. IMO that better fits our usage because we expressly want to sign with this key.